src/operator-manual/metrics/01-quickstart.md

```diff
@@ -0,0 +1,235 @@
+need set this values if you don't want to change anything else.
+
+We can install the helm chart:
+
+```console
+helm install --wait --create-namespace --namespace kubewarden kubewarden-controller kubewarden/kubewarden-controller
+```
+
+This leads to the creation of the `default` instance of `PolicyServer`:
+
+```console
+kubectl get policyservers.policies.kubewarden.io
+NAME      AGE
+default   3m7s
+```
+
+By default, this policy server will have metrics enabled.
+
+## Enforcing a policy
+
+We will start by deploying the [safe-labels](https://github.com/kubewarden/safe-labels-policy)
+policy.
+
+We want the policy to be enforced only inside of Namespaces that have a
+label `environment` with value `production`.
+
+Let's create a Namespace that has such a label:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: team-beta-prod
+  labels:
+    environment: production
+EOF
+```
+
+Next, let's define a ClusterAdmissionPolicy:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: ClusterAdmissionPolicy
+metadata:
+  name: mandatory-owner
+spec:
+  module: registry://ghcr.io/kubewarden/policies/safe-labels:v0.1.6
+  settings:
+    mandatory_labels:
+    - owner
+  rules:
+    - apiGroups:
+        - "*"
+      apiVersions:
+        - "*"
+      resources:
+        - "*"
+      operations:
+        - CREATE
+        - UPDATE
+  namespaceSelector:
+    matchExpressions:
+    - key: environment
+      operator: In
+      values: ["production"]
+  mutating: false
+EOF
+```
+
+We can wait for the policy to be active in this way:
+
+```console
+kubectl wait --for=condition=PolicyActive clusteradmissionpolicy/mandatory-owner
+```
+
+Once the policy is active, we can try it out in this way:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  namespace: team-beta-prod
+  labels:
+    owner: some-owner
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 0
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+EOF
+```
+
+This Deployment object will be created because it doesn't violate the policy.
+
+On the other hand, this Deployment will be blocked by the policy:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment-without-labels
+  namespace: team-beta-prod
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 0
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+EOF
+```
+
+One of the metrics that the `PolicyServer` exposes is the number or policy evaluations. We can check
```

Comment:
I would create a new section like `Accessing Prometheus` with this information,
plus another section about `Accessing Grafana`.

They don't have to dig into the details, just provide the URLs (and eventual `kubectl port-forward` command) required to access them